### ISSUE TITLE
Mention users in shared documents

### ISSUE THREAD

**Issue by MelBr01** [CollaboraOnline/online#7923]
### Describe the Bug
When trying to mention users using @ in a document in a shared folder, I get an empty box with the text "No search results found!".

### Steps to Reproduce
1. Create a document in a shared folder.
2. Type @ and the first letter of a user.
3. The above box appears with no search results found.

### Expected Behavior
A list of users that can be mentioned.

### Actual Behavior
A box with no search results found.

### Screenshots
A user with the first letter R has access to the document. A lower case r produces the same results.
<img width="235" alt="image" src="https://github.com/CollaboraOnline/online/assets/149402562/8d49b9a2-b780-4a26-9ac2-af173d72d20e">

### Desktop
(Please complete the following information)
 - **Collabora version:** 23.05.5.4
 - **OS and version:** Windows 11
 - **Browser and version:** Edge 120.0.2210.91 (Official build) (64-bit)

**Comment by eszkadev** [comment]
Which storage integrator do you use? (eg. Nextcloud? which version)

**Comment by MelBr01** [comment]
Nextcloud 27.1.5

**Comment by timur-g** [comment]
There was a related change for 23.05.6.4, please test if that fixes for you.

**Comment by MelBr01** [comment]
No, unfortunately not. Still the same problem in Libra office and in Word....

**Comment by timur-g** [comment]
I do not understand this for LibreOffice and Word. Mention should be possible using Collabora Online 23.05.6 with NextCloud users.

**Comment by MelBr01** [comment]
Unfortunately it is not. Whenever I try to add a mention the same error occurs: 
<img width="321" alt="image" src="https://github.com/CollaboraOnline/online/assets/149402562/bed7fb75-7e61-49a2-8e3c-82430245741d">

**Comment by sebastiaanveld** [comment]
Tested this with NC 27.1.5 and it does also not work for me with CODE 23.05.6.4.

COOLWSD version: 23.05.6.4 (git hash: aff0ccf (E))
LOKit version: Collabora Office 23.05.6.4 (git hash: 058f614)
Served by: Debian GNU/Linux 12 (bookworm)
Server ID: 0c34bb0c

![image](https://github.com/CollaboraOnline/online/assets/37268999/0e0361e7-509c-4fa3-9a23-f259214e4c08)

**Comment by timur-g** [comment]
Please see if double @@mention works for you.

**Comment by timur-g** [comment]
This should work in 24.04.9, please test. If not, please explain.
